media-gfx/blender/blender-2.78.ebuild

```diff
@@ -0,0 +1,299 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python3_5 )
```

Comment:
no moar py3.4? Fine with me, you just won't be able to stabilise it any time soon...